CODEOWNERS

```diff
@@ -1,8 +1,25 @@
-# Default rules
+# Website Development
+* @nodejs/nodejs-website
 
-* @nodejs/website
+# Infrastructure
+.github @nodejs/web-infra
+.husky @nodejs/web-infra
 
-# Node.js Release Blog Posts
+# Dependencies
+package.json @nodejs/web-infra
```

Comment:
Should this be infra? I feel like a lot of dependency changes would fall under day-to-day development of new features, maintenance, etc.?

Reply:
Fair enough